src/backend/distributed/planner/distributed_planner.c

```diff
@@ -1239,8 +1239,7 @@ RemoteScanRangeTableEntry(List *columnNameList)
 {
 	RangeTblEntry *remoteScanRangeTableEntry = makeNode(RangeTblEntry);
 
-	/* we use RTE_VALUES for custom scan because we can't look up relation */
-	remoteScanRangeTableEntry->rtekind = RTE_VALUES;
```

Comment:
@metdos According to git blame you wrote this original comment 3 years ago. I made this change to make the explain plan work in this PR and it doesn't seem to break anything. Is there anything that I might be missing or do you think this is indeed a safe change.

Reply:
Is this still necessary?

Reply:
undid this change